tools/dialignr/dialignr.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="dialignr" name="DIAlignR" version="@WRAPPER_VERSION@">
+    <description>for retention time alignment of targeted mass spectrometric data</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">bioconductor-dialignr</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    #import re
+
+    mkdir -p ./data/mzml &&
+    mkdir -p ./data/osw &&
+
+    #for $mzml in $input_mzml_files:
+        #set $file_name_mzml = re.sub('[^\w\-_.]', '_', str($mzml.element_identifier))
+        echo '$file_name_mzml' >> ./runs.txt &&
+        ln -s '$mzml' './data/mzml/${file_name_mzml}.chrom.mzML' &&
+    #end for
+
+    ln -s '$input_osw_file' './data/osw/merged.osw' &&
+
+    Rscript --vanilla '${__tool_directory__}/dialignr.R' `pwd`
+    ]]></command>
+    <inputs>
+        <param name="input_mzml_files" type="data" format="mzml" multiple="true" label="MZML file(s)" help="Sample file(s) in .MZML format." />
+        <param name="input_osw_file" type="data" format="osw" multiple="false" label="OSW file" help="Sample file in .OSW format." />
+    </inputs>
+    <outputs>
+        <data name="out_aligned_targeted_runs" format="csv" label="${tool.name} on ${on_string}: alignedTargetedRuns.csv" from_work_dir="alignedTargetedRuns.csv" />
```

Comment:
can we convert this file to tsv in R ... CSV are not that useful in Galaxy

Reply:
Done!